src/lib.rs

```diff
@@ -176,30 +177,30 @@ pub fn builder() -> Builder {
 }
 
 impl Builder {
-    /// Set number of submission queue entries in uring.
-    ///
-    /// The kernel will ensure it uses a power of two and will round this up if necessary.
-    /// The kernel requires the number of completion queue entries to be larger than
-    /// the submission queue entries so generally will double the sq entries count.
+    /// Sets the number of Submission Queue entries in uring.
     ///
-    /// The caller can specify even a larger cq entries count by using the uring_builder
-    /// as shown in the start example below.
-    pub fn entries(&mut self, e: u32) -> &mut Self {
-        self.entries = e;
+    /// The default value is 256.
+    /// The kernel requires the number of submission queue entries to be a power of two,
+    /// and that it be less than the number of completion queue entries.
+    /// This function will adjust the `cq_entries` value to be at least 2 times `sq_entries`
+    pub fn entries(&mut self, sqe: u32) -> &mut Self {
+        self.entries = sqe;
         self
     }
 
-    /// Replace the default io_uring Builder. This allows the caller to craft the io_uring Builder
-    /// using the io_uring crate's Builder API.
+    /// Replaces the default [`io_uring::Builder`], which controls the settings for the
+    /// inner `io_uring` API.
+    ///
+    /// This interface offers many more settings and parameters,
+    /// but requires directly depending upon the `io_uring` crate.
```

Comment:
There is the `uring_builder` method which does return io_uring::Builder.  I am assuming that io_uring is re-exported then.  

With that in mind, maybe the right thing to do is to eliminate the tokio_uring::Builder method since it isn't really a Builder, but a convenience wrapper for supplying the SQ Entries parameter to `io_uring::Builder::build()`

We can discuss that in #204

Reply:
yes. Two levels of builder. Not idiomatic at all.